pkg/virt-handler/node-labeller/node_labeller_test.go

```diff
@@ -127,19 +127,25 @@ var _ = Describe("Node-labeller ", func() {
 	})
 
 	It("should add host cpu model label", func() {
-		expectNodePatch(kubevirtv1.HostModelCPULabel)
 		res := nlController.execute()
+		expectNodePatch(kubevirtv1.HostModelCPULabel)
```

Comment:
why would you need to change this test (and the one bellow)? Also it does not seem correct to setup the expectations **after** calling `execute()` (this applies to all the tests here).